Brewfile

```diff
@@ -1,4 +1,4 @@
-brew "bufbuild/buf/buf"
```

Comment:
https://buf.build/docs/installation still advertises to use `bufbuild/buf/buf`.

What is likely missing is `tap "bufbuild/buf"` in this file at the top.